src/cmd/build.js

```diff
@@ -78,7 +79,8 @@ export type PackageCreatorFn =
     (params: PackageCreatorParams) => Promise<ExtensionBuildResult>;
 
 async function defaultPackageCreator(
-  {manifestData, sourceDir, fileFilter, artifactsDir}: PackageCreatorParams
+  {manifestData, sourceDir, fileFilter, artifactsDir,
+    showReadyMessage}: PackageCreatorParams
```

Comment:
oops, I completely missed this one. This is also formatted inconsistently because the block is only partially broken into the next line. An easy fix would be:
````javascript
async function defaultPackageCreator({
  manifestData, sourceDir, fileFilter, artifactsDir, showReadyMessage
}: PackageCreatorParams): Promise<ExtensionBuildResult> {
````